pkg/util/defaults/defaults.go

```diff
@@ -44,5 +44,10 @@ const (
 	installDefaultKamelets = true
 )
 
+// Maps architecture variants for common base images
+var baseImageArchMapping = map[string]string{
+	"eclipse-temurin:17-arm64": "eclipse-temurin:17@sha256:a90cec83bb9b9ab19a63e377b20c3aa4e076b16c52d36e83c62c451b6d034e22",
```

Comment:
We cannot hardcode this value. We need to drive this information from configuration as it happens for the other variables.